bin/postgres-ha/bootstrap-postgres-ha.sh

```diff
@@ -158,7 +158,18 @@ then
     echo_info "Database manually started"
 
     echo_info "Waiting to reach a consistent state"
-    test_server "postgres" "${PGHOST}" "${PGHA_PG_PORT}" "postgres"
+    until pg_isready --dbname="postgres" --host="${PGHOST}" --port="${PGHA_PG_PORT}" --username="postgres"
+    do
+        "Database has not reached a consistent state, sleeping..."
+        # sleep to give recovery a chance to complete
+        sleep 30
```

Comment:
@andrewlecuyer Though I think I had suggested this value yesterday, I wonder if this is too long, e.g. if this is the only DB that can be brought up in the area? Maybe we should go back to 5....even if it's a long time recovery. I think the cost is low.

Reply:
Sleep is now back to 5 seconds.